### ISSUE TITLE
Keyboard navigation not working correctly when scale step higher than 1

### ISSUE THREAD

**Issue by kevindoherty30** [adaptlearning/adapt-contrib-slider#187]
### Subject of the issue
Keyboard navigation does not work correctly when scale step is higher than 1

### Your environment
* f/w v5.28.11
* Slider v4.3.5 & Confidence Slider v5.4.4
* all browsers

### Steps to reproduce

- Create a course
- Add a slider component
- Set the scale to go from 0 to 5
- Set the scale step to 1
- Duplicate the component
- Edit the new copy of slider
- Change the scale to go from 0 to 50
- Set the scale step to 10
- Save and preview
- Tab to the slider with a scale step of 1 - once on the slider scale use the arrow keys to move it
- Make a selection and then tab to submit and hit enter
- Slider will be submitted as expected
- Tab to the slider with the scale step of 10 once on the slider scale use the arrow keys to move it

### Expected behaviour

- The slider should move up and down depending on whether the up/down or left/right arrow buttons on keyboard are used

### Actual behaviour

- The slider doesn’t move

### Additional Informaton:

- Works when scale step is set to integers of 1 but for some reason is not working with anything higher than that. When set to a scale step of 2 it is possible to move up the scale but not down. As soon as 3 or more it does not move.
